src/content/reference/cli.mdx

```diff
@@ -77,9 +77,17 @@ The following flags can be used (very similar to `docker build`):
 | _--secret_       |  (list)  | Secret files exposed to the build. Format: `id=mysecret,src=/local/secret` |
 | _--export-cache_ | (string) | Image tag for exported cache when build (optional)                         |
 | _--cache-from_   |  (list)  | List of cache source images (optional)                                     |
+| _--platform_     | (string) | Specify which platform to build the container image for (optional)                                     |
```

Comment:
Can we include the allowed values?  (Or at least the most common?)

Reply:
Could you mention about what are the allowed values you're mentioning here? Or any reference if they have been documented somewhere?

Reply:
I don't think we have them documented anywhere. @pchico83 which platform combinations do we support?

Reply:
I think we can go with the platforms I've mentioned in the example -  x86_64 and ARM-based platforms. Wdyt?

Reply:
Buildkit support AMD 64-bit, Arm 64-bit, and Armv7 architectures

Reply:
I've already provided examples for AMD 64 and ARMv7. Do we need to make any changes further?